client/src/features.js

```diff
@@ -1,5 +1,5 @@
-const cfp = true;
-const voting = false;
+const cfp = false;
```

Comment:
you need to disable (or remove) the form at `/cfp`. If someone tries to hack by calling the JS function that submits or sends an HTTP request - that's ok, we'll live with that, but the form should be removed.

Reply:
now there a message "Reversim Summit 2018 - CFP is closed!"